AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/JoinCallConfig.java

```diff
@@ -16,14 +16,14 @@ public JoinCallConfig(
             final boolean isMicrophoneMuted,
             final boolean isCameraOn,
             final String displayName) {
-        this.groupId = groupId;
+        this.joinId = groupId;
```

Comment:
Do we still have a `groupId` somewhere?

Reply:
Thanks Wayne. also changed function input parameters name.